docs/user-guide/ze-usage.md

```diff
@@ -0,0 +1,287 @@
+
+* **View Unix System Services (USS) files**: You can view multiple USS files simultaneously.
+* **Rename USS files**: You can rename specified USS files.
+* **Download, edit, and upload existing USS files**: You can instantly pull USS files from the mainframe, edit them and upload back.
+* **Create and delete USS files and directories**: Enables you to easily create and delete both USS files and directories.
+* **View and access multiple profiles simultaneously**: Enables to work with USS files from multiple profiles.
+
+#### View Unix System Services (USS) files
+
+1. Navigate to the explorer tree.
+2. Open the **Unix System Services (USS)** bar.
+3. Select the profile that you want to filter.
+4. Click the **Search Unix System Services (USS) by Entering a Path** magnifying glass.
+5. From the drop-down, enter the path that you want as the root of your displayed tree.  
+  All child files and directories of that root file are displayed in the explorer tree.
+
+  **Note:** You will not be able to expand directories or files that you are not authorised for.
+
+   <img src="../images/ze/ZE-path.gif" alt="Enter Path">
+
+#### Refresh the list of files
+
+1. Navigate to the explorer tree.
+2. Click **Refresh All** button on the right of the **Unix System Services (USS)** explorer bar as illustrated by the following screen:
+
+<img src="../images/ze/ZE-refreshUSS.gif" alt="Refresh All">
+
+#### Rename USS files
+
+1. Navigate to the explorer tree.
+2. Open the **USS** bar.
+3. Select a USS file you want to rename.
+4. Right-click the USS file and select the **Rename USS file** option.
+5. Change the name of the USS file.
+
+#### Download, edit, and upload an existing file
+
+1. Click the file that you want to download.
+
+    **Note:** To view the files within a directory, click the directory to expand the tree.
+
+    The file displays in the text editor window of VSC.
+
+    **Note:** If you define file associations with syntax coloring, the suffix of your file will be marked up.
+
+2. Edit the document.
+3. Type Ctrl+S or Command+S (OSx) to save the file
+
+Your file is uploaded.  
+
+<img src="../images/ze/ZE-editUSS.gif" alt="Edit">
+
+#### Creating and deleting files and directories
+
+#### Create a directory
+
+1. Navigate to the explorer tree.
+2. Open the **Unix System Services (USS)** bar.
+3. Select a directory that you want to add the new directory to.
+4. Select the **Create directory** button and specify the directory name.
+   The directory is created.
+
+#### Create a file
+
+1. Navigate to the explorer tree.
+2. Open the **Unix System Services (USS)** bar.
+3. Select a directory that you want to add the new file to.
+4. Select the **Create file** button and specify the file name.
+   The file is created.
+
+#### Delete a file
+
+1. Navigate to the explorer tree.
+2. Open the **Unix System Services (USS)** bar.
+3. Select a file you want to remove.
+4. Select the **Delete** button and press *Yes** to confirm.
```

Comment:
Press? I see this several times. You also use "click". Can we be consistent?

Reply:
yes, you are right here. Press refers to keyboard keys.